sharding-orchestration/sharding-orchestration-core/src/main/java/io/shardingsphere/orchestration/internal/registry/config/node/ConfigurationNode.java

```diff
@@ -108,4 +111,20 @@ private String getFullPath(final String schemaName, final String node) {
     private String getFullPath(final String node) {
         return Joiner.on("/").join("", name, ROOT, node);
     }
+    
+    /**
+     * Get schema Name.
+     * 
+     * @param configNodeFullPath config node full path
```

Comment:
Change `config node full path` to `configuration node full path`, do not use abbreviate with javadoc


.